test/e2e/grpc_test.go

```diff
@@ -57,9 +57,16 @@ func dial(host, domain string) (*grpc.ClientConn, error) {
 	if host != domain {
 		// The host to connect and the domain accepted differ.
 		// We need to do grpc.WithAuthority(...) here.
+
+		// We need to remove the port part in the domain to use it as the authority
```

Comment:
Why do we even force-add the port a block above then?

Reply:
I don't really know :) 

I've been taking a look at the gRPC "specification" to see if the authority field should or should not specify the port: all the examples I've found don't add the port to the authority field... 

We are toying with a custom knative ingress, and we have been bitten by this. We can fix it by using a regex when matching the host (adding some optional part like knative does with istio ingress), or fixing the authority here, and hoping this is the "standard".

Reply:
There was some discussion about this. @tanzeeb , I think, did the original HTTP2 suppport, he might have more context.

Reply:
Context: https://github.com/knative/serving/pull/3398/files#r263902617

Reply:
Ok, then we can drop this PR and we will implement support for authority + port. 

Sorry and thank you!